modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -34,10 +31,8 @@ $ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
 | Show help.
 |===
 
-[id="cli-reference-flows-capture-options_{context}"]
 == Flows capture options
-Flows capture has mandatory commands as well as additional options, such as enabling extra features about packet drops, DNS latencies, Round-trip time, and filtering.
-
+Flow capture has mandatory commands as well as additional options, such as enabling extra features about packet drops, DNS latencies, Round-trip time, and filtering.
 .`oc netobserv flows` syntax
```

Comment:
```suggestion
Flow capture has mandatory commands as well as additional options, such as enabling extra features about packet drops, DNS latencies, Round-trip time, and filtering.

.`oc netobserv flows` syntax
```

Reply:
Need a return here, otherwise the heading rendering is off: 
![image](https://github.com/user-attachments/assets/232f6721-b811-4d85-ab77-43688cdcced6)